### PR TITLE
OH2-230: Add the max length property and Capital letter on description

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OH2-230.

### CODE REVIEW

File: src/main/java/org/isf/disease/dto/DiseaseDTO.java [hunk 1]
```diff
@@ -32,32 +32,32 @@
 public class DiseaseDTO {
 
 	@NotNull
-	@Schema(description = "Disease code", example = "99")
+	@Schema(description = "Disease code", example = "99", maxLength = 10)
 	private String code;
 
 	@NotNull
-	@Schema(description = "Disease description")
+	@Schema(description = "Disease description", maxLength = 50)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/disease/dto/DiseaseDTO.java:39]: the next line looks like the indentation is wrong.

File: src/main/java/org/isf/distype/dto/DiseaseTypeDTO.java [hunk 2]
```diff
@@ -29,11 +29,11 @@
 public class DiseaseTypeDTO {
 
 	@NotNull
-	@Schema(description = "Disease type code")
+	@Schema(description = "Disease type code", maxLength = 2)
 	private String code;
 
 	@NotNull
-	@Schema(description = "Disease type description")
+	@Schema(description = "Disease type description", maxLength = 110)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/distype/dto/DiseaseTypeDTO.java:36]: How did you come up with 110?  Seems like an odd number to me.
BTW, the next line is indented incorrectly.

File: src/main/java/org/isf/sms/dto/SmsDTO.java [hunk 3]
```diff
@@ -41,25 +41,25 @@ public class SmsDTO {
 	private LocalDateTime smsDateSched;
 	
 	@NotNull
-	@Schema(description = "SMS target phone number", example="+237671302313")
+	@Schema(description = "SMS target phone number", example="+237671302313", maxLength = 45)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/sms/dto/SmsDTO.java:44]: All other phone numbers are 50 characters long.

File: src/main/java/org/isf/supplier/dto/SupplierDTO.java [hunk 4]
```diff
@@ -32,25 +32,25 @@ public class SupplierDTO {
...
-	@Schema(description="The supplier's address", example = "25 Rue Ministre, Dschang")
+	@Schema(description="The supplier's address", example = "25 Rue Ministre, Dschang", maxLength = 150)
 	private String supAddress;
 	
-	@Schema(description="The supplier's tax code", example = "5221")
+	@Schema(description="The supplier's tax code", example = "5221", maxLength = 50)
 	private String supTaxcode;
 	
-	@Schema(description="The supplier's phone", example = "+237654120145")
+	@Schema(description="The supplier's phone", example = "+237654120145", maxLength = 20)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/supplier/dto/SupplierDTO.java:44]: All other phone numbers are either 45 or 50 characters.

File: src/main/java/org/isf/supplier/dto/SupplierDTO.java [hunk 5]
```diff
@@ -32,25 +32,25 @@ public class SupplierDTO {
...
 	private String supPhone;
 	
-	@Schema(description="The supplier's fax number", example = "+237654120145")
+	@Schema(description="The supplier's fax number", example = "+237654120145", maxLength = 20)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/supplier/dto/SupplierDTO.java:47]: Shouldn't this be the same length as phone numbers?

File: src/main/java/org/isf/distype/dto/DiseaseTypeDTO.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/distype/dto/DiseaseTypeDTO.java:36]: It is from the DDL: 

DCL_ID_A | char(2) PK
-- | --
DCL_DESC | varchar(110)
DCL_CREATED_BY | varchar(50)
DCL_CREATED_DATE | datetime
DCL_LAST_MODIFIED_BY | varchar(50)
DCL_LAST_MODIFIED_DATE | datetime
DCL_ACTIVE | tinyint(1)

File: src/main/java/org/isf/lab/dto/LaboratoryDTO.java [hunk 6]
```diff
@@ -53,10 +53,10 @@ public class LaboratoryDTO {
 	@Schema(description = "Laboratory Result")
 	private String result;
 
-	@Schema(description = "lock", example = "0")
+	@Schema(description = "Lock", example = "0")
 	private int lock;
 
-	@Schema(description = "Laboratory Note", example = "Note by laboratorist")
+	@Schema(description = "Laboratory Note", example = "Note by laboratorist", maxLength = 255)
```
[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/lab/dto/LaboratoryDTO.java:59]: same as above:

LAB_ID | int(11) AI PK
-- | --
LAB_EXA_ID_A | varchar(10)
LAB_DATE | datetime
LAB_RES | varchar(50)
LAB_NOTE | varchar(255)
LAB_PAT_ID | int(11)
... | ...

File: src/main/java/org/isf/supplier/dto/SupplierDTO.java (hunk 5 above)
Comment (mwithi) [reply src/main/java/org/isf/supplier/dto/SupplierDTO.java:47]: I think we should make all the same, starting by the DB...

Comment (ArnaudFonzam) [reply src/main/java/org/isf/supplier/dto/SupplierDTO.java:47]: supplier
I use the lenght specify in the database

File: src/main/java/org/isf/sms/dto/SmsDTO.java (hunk 3 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/sms/dto/SmsDTO.java:44]: sms
all those size are take in the database.

File: src/main/java/org/isf/supplier/dto/SupplierDTO.java (hunk 5 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/supplier/dto/SupplierDTO.java:47]: > I think we should make all the same, starting by the DB... not in this PR.
> 
> 20 chars sound reasonable and enough, wdyt?
I agree with you

File: src/main/java/org/isf/disease/dto/DiseaseDTO.java (hunk 1 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/disease/dto/DiseaseDTO.java:39]: please for this i don't see the wrong indentation

Comment (dbmalkovsky) [reply src/main/java/org/isf/disease/dto/DiseaseDTO.java:39]: It is not line 39 but line 40 that looks wrong; I could not put a comment on that line and that is the reason I wrote:  "the next line looks....."

File: src/main/java/org/isf/disease/dto/DiseaseDTO.java [hunk 7]
```diff
@@ -32,32 +32,32 @@
...
     private String description;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/disease/dto/DiseaseDTO.java:40]: Here is the that looks like the indention is wrong;   there are other occurrences too that I haven't bothered to mark.

Comment (ArnaudFonzam) [reply src/main/java/org/isf/disease/dto/DiseaseDTO.java]: please how many tab space do you use in you IDE please.

File: src/main/java/org/isf/disease/dto/DiseaseDTO.java (hunk 1 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/disease/dto/DiseaseDTO.java:39]: OKay I think it is okay now

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**